### PR TITLE
feat: content split; local + deploy previews 

### PR DESCRIPTION
**Pull request by thiskevinwang**
# Description

The ETL'ing of this content blocks
- https://github.com/hashicorp/terraform-website/pull/2283

### CODE REVIEW

**Comment by BRKalow** [comment]
Wanna add a `gitignore` with `node_modules` and `website-perview` specified?

**Review by laurapacilio — APPROVED** [review]

File: website/docs/plugin/sdkv2/testing/testing-api.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (bflad) [inline website/docs/plugin/sdkv2/testing/testing-api.mdx:1]: Not for this pull request, but we should drop this empty page. 👍

File: website/docs/plugin/sdkv2/testing/testing-patterns.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (bflad) [inline website/docs/plugin/sdkv2/testing/testing-patterns.mdx:1]: Not for this pull request, but we should drop this empty page. 👍

**Review by bflad — APPROVED** [review]
Other than needing a `.gitignore` file added, looks good to me 🚀

**Pull request merged**